content/al/0018-matriz-mudanca-de-coordenadas.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Matriz de mudança de coordenadas
+description: >-
+  Como obter uma matriz de mudança de coordenadas
+path: /al/matriz-mudanca-de-coordenadas
+type: content
+---
+
+# Matriz de mudança de coordenadas
+
+```toc
+
+```
+
+## Matriz de mudança de coordenadas
+
+:::tip[Definição]
+
+Sejam $B_1=\{v_1,...,v_n\}$ e $B_2=\{w_1,...,w_n\}$, duas bases ordenadas de um espaço vetorial $V$,
+a matriz $S_{B_1 \to B_2} = (s_{ij})_{n\times n}$, cujas colunas são as coordenadas dos vetores de $B_1$ escritos na base $B_2$,
+chama-se matriz de mudança de coordenadas de $B_1$ para $B_2$.
+
+:::
+
+:::tip[Teoremas]
+
+A matriz $S_{B_1 \to B_2}$ satisfaz a igualdade:
+
+$$
+S_{B_1 \to B_2} [u]_{B_1} = [u]_{B_2}
+$$
+
+**A matriz de mudança de coordenadas permite assim calcular as coordenadas de um qualquer vetor $u \in V$ na base $B_2$ desde que sejam conhecidas as suas coordenadas na base $B_1$.**
+
+Da igualdade acima podemos inferir que:
+
+$$
+S_{B_1 \to B_2} [u]_{B_1} = [u]_{B_2} \Leftrightarrow  S_{B_1 \to B_2}^{-1} [u]_{B_2} = [u]_{B_1}
+$$
+
+ou seja, que
+
+$$
+S_{B_1 \to B_2}^{-1} = S_{B_2 \to B_1}
+$$
+
+Menos óbvia mas também válida é a seguinte igualdade:
+
+$$
+S_{B_1 \to B_3} = S_{B_2 \to B_3}S_{B_1 \to B_2}
+$$
+
+Sendo aplicada a matriz de mudança de coordenadas $S_{B_2 \to B_3}$ após a matriz de mudança de coordenadas $S_{B_1 \to B_2}$,
+podemos obter a matriz de mudança de coordenadas $S_{B_1 \to B_3}$.
+
+:::
+
+## Construção da matriz de mudança de coordenadas
+
+:::tip[Resolução]
```

Comment:
I don't think this should be a container at all, just regular text

Reply:
Agreed